hc/api/tests/test_check_model.py

```diff
@@ -36,3 +40,9 @@ def test_paused_check_is_not_in_grace_period(self):
         self.assertFalse(check.in_grace_period())
 
     ### Test that when a new check is created, it is not in the grace period
+
+    def test_new_not_in_grace_period(self):
```

Comment:
method is not cescriptive enough perhaps use
test_new_check_not_in_grace_period(self):